game/src/debug/select_roads.rs

```diff
@@ -0,0 +1,121 @@
+use std::collections::BTreeSet;
```

Comment:
This is the old `edit/bulk.rs` with some stuff removed

Reply:
Just curious, do you have a use case for these exports?

Reply:
Export to [shared-row](https://github.com/d-wasserman/shared-row/) was for a prototype integration with some of David's 3D viz tools; that stalled out. Export to streetmix format is for #386 and for Kieran to try out stuff with 3dst. And the export list of roads was for.. probably some manual OSM tagging process. I think all 3 are worth keeping around, but hiding.